mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAOTest.java

```diff
@@ -188,4 +197,133 @@ private MessageWithoutAttachment toMessage(CompletableFuture<Stream<CassandraMes
             .orElseThrow(() -> new IllegalStateException("Collection is not supposed to be empty"));
     }
 
+    @Test
+    public void retrieveAllMessageIdAttachmentIdsShouldReturnEmptyWhenNone() {
+        Stream<MessageIdAttachmentIds> actual = testee.retrieveAllMessageIdAttachmentIds().join();
+        
+        assertThat(actual).isEmpty();
+    }
+
+    @Test
+    public void retrieveAllMessageIdAttachmentIdsShouldReturnOneWhenStored() throws Exception {
+        //Given
+        MessageAttachment attachment = MessageAttachment.builder()
+                .attachment(Attachment.builder()
```

Comment:
ident?

Reply:
Please Indent is 4 space, not 8! Check your IDE settings.